functions-methods.js

```diff
@@ -33,4 +50,17 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+    if (email.slice(-1) ===".") {
+        return email = "false";
+    } else if (email.includes("@") && !email.includes(",")) {
+    return email = "true";
+} else {
+        return email = "false";
+    }
```

Comment:
Je zou het nog iets kunnen simplificeren:
```suggestion
    return email.slice(-1) !== "." && email.includes("@") && !email.includes(",");
```